nginx-controller/nginx/configurator.go

```diff
@@ -380,6 +389,30 @@ func (cnf *Configurator) createConfig(ingEx *IngressEx) Config {
 		ingCfg.Ports = ports
 	}
 
+	ingCfg.Enabled = true
+	if listenAddress, exists := ingEx.Ingress.Annotations["nginx.org/listen-address"]; exists {
+		ingCfg.Enabled = false
+
+		addrs, err := net.InterfaceAddrs()
+		if err != nil {
+			glog.Errorf("unable to list interface addresses: %s", err)
+		} else {
+			listenIP := net.ParseIP(listenAddress)
+			if err != nil {
```

Comment:
this `err != nil` is very weird since net.ParseIP doesn't return error. Actually, else-body will be not executed ever.

Reply:
Actually, else-body will be executed, because `err == nil` in this case (coming from `net.InterfaceAddrs()` call). Anyway, it's leftover from another "parse IP" function call, removed now.